core/arch/arch.h

```diff
@@ -1575,9 +1575,6 @@ cache_pc
 get_direct_exit_target(dcontext_t *dcontext, uint flags);
 
 #ifdef AARCHXX
-size_t
-get_fcache_return_tls_offs(dcontext_t *dcontext, uint flags);
-
```

Comment:
Since this is ARM-local, it's probably fine to clean it from  AArch64. But I'm curious what motivated this change: did this show up as some sort of error in RISC-V?

Reply:
With this change, I can just change the line above to `#if defined(AARCHXX) || defined(RISCV64)`, and no need to implement an unused function.